### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by masterT**
- Add secret `SLACK_CHANNEL`.
- Add `README.md` file.

### CODE REVIEW

File: .github/workflows/scraper.yml [hunk 1]
```diff
@@ -19,12 +19,13 @@ jobs:
...
     - name: Updating the repository
       run: git push origin HEAD:${{ github.ref }}
-    
+
     - name: Notify Slack channel
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
       run: |
-        curl -X POST -d "{\"channel\":\"CMR0875M3\",\"text\":\"The CSGames :csgames: website was updated. See https://github.com/${{ github.repository }}/commit/$(git rev-parse HEAD)\"}" https://slack.com/api/chat.postMessage -H "Authorization: Bearer $SLACK_BOT_TOKEN" -H "Content-type: application/json"
```
Comment (afrigon) [inline .github/workflows/scraper.yml:32]: slack channel is not sensitive informations. I don't even have access to secrets on this repo 😢

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Envoie un message dans un "channel" Slack lorsque le site web des [CSGames](https://csgames.org/) change pour l'année courante.
+
+Ce projet utilise les [GitHub Actions](https://help.github.com/en/actions) pour automatiser le processus.
+
+## Configuration
+
+Ajouter les secrets suivant au dépôt GitHub via dans la [section Secrets des Settings](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets):
+- `SLACK_BOT_TOKEN` : le jeton Slack permettant de communiquer dans un "workspace" Slack.
+- `SLACK_CHANNEL` : l'identifiant du "channel" Slack.
```
Comment (afrigon) [inline README.md]: we're doing french now ?

Comment (masterT) [reply README.md]: Switched to English!

File: .github/workflows/scraper.yml (hunk 1 above)
Comment (masterT) [reply .github/workflows/scraper.yml:32]: Change for ENV variable in the workflow.

**Review by afrigon — APPROVED** [review]

**Pull request merged**